Alexey/Files/PDF_generator/tests/test_pdf_generator.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from Alexey.Files.PDF_generator.main.PDF_generator import txt_file_reader, pdf_file_writer
+import fpdf
+
+class TestPdfGenerator(unittest.TestCase):
```

Comment:
Add more tests
1) When one big single line is converted
2) When two pages should be used 
Use separate test-resource folder to save files